backend/pretest-script.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# set node env to test, if windows use set instead of export
+if [[ "$OSTYPE" == "msys" ]]; then
+  # For Windows CMD: set "NODE_ENV=test"
+  # For PowerShell: $env:NODE_ENV="test"
+  set "NODE_ENV=test"
+  $env:NODE_ENV="test"
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing '='. Bad assignment or comparison?

(SC2276)